rosidl_typesupport_opensplice_c/rosidl_typesupport_opensplice_c/__init__.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from rosidl_cmake import convert_camel_case_to_lower_case_underscore
+from rosidl_cmake import expand_template
+from rosidl_cmake import extract_message_types
+from rosidl_cmake import get_newest_modification_time
+from rosidl_parser import parse_message_file
+# from rosidl_parser import parse_service_file
+from rosidl_parser import validate_field_types
+
+
+def generate_typesupport_opensplice_c(args):
+    template_dir = args['template_dir']
+    mapping_msgs = {
+        # os.path.join(template_dir, 'msg__type_support.hpp.template'): '%s__type_support.hpp',
```

Comment:
AFAIK, there are exactly three places where there is commented code and one empty file that are all related to services. I think it would be more work to have to dig out the diff of what needs to be added where in the event that we need to add some service specific code, so I decided to leave it.